pymc3/distributions/continuous.py

```diff
@@ -203,26 +201,22 @@ class Normal(Continuous):
         Precision (tau > 0).
     """
 
+    rng = stats.norm.rvs
```

Comment:
We should wrap the SciPy's `rvs` function to use PyMC argument names. SciPy uses generic `loc` and `scale` names, which is confusing. This has been done for some of the distributions I see, but not all.